provision/acc_provision/acc_provision.py

```diff
@@ -363,7 +365,19 @@ def normalize_cidr(cidr):
 
 
 def config_adjust(args, config, prov_apic, no_random):
-    system_id = config["aci_config"]["system_id"]
+    if is_calico_flavor(config["flavor"]):
+        l3out_name = ""
+        if not config["aci_config"]["cluster_l3out"].get("name"):
+            vlan_id = config["aci_config"]["cluster_l3out"]["svi"]["vlan_id"]
+            l3out_name = "calico-l3out-vlan-%s" % vlan_id
+            config["aci_config"]["cluster_l3out"]["name"] = l3out_name
+        l3out_name = config["aci_config"]["cluster_l3out"]["name"]
+        system_id = "calico-%s" % l3out_name if "calico" not in l3out_name else l3out_name
```

Comment:
this should be config["flavor"]-%s and so on

Reply:
We want the "calico" prefix to be present regardless of the specific calico flavor (note that we will will have multiple calico flavors depending on the tigera operator version  and/or if the flavor is to be consumed by CKO).

Reply:
those names could all be required to have calico in them.. but yeah functionality wise looks good. This was a nit.